features/step_definitions/admin_steps.rb

```diff
@@ -307,6 +307,12 @@ def update_settings(settings)
 
 ### THEN
 
+Then(/^I should see the last login time for "(.*?)"/) do |user|
+  u = User.find_by_login(user)
+  assert Time.now - u.current_sign_in_at < 2.minutes
```

Comment:
We should freeze time in the test for the step `When I am logged in as "new_user"` and check if "Last Activity" shows that point of frozen time.